### PR TITLE
add ddt

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@jakirkham Ready for review

**Comment by CJ-Wright** [comment]
@conda-forge/core Ready for review

File: recipes/ddt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+#    - codecov
+#    - coverage
```
Comment (scopatz) [inline recipes/ddt/meta.yaml]: please remove comments

Comment (CJ-Wright) [inline recipes/ddt/meta.yaml]: Totally forgot about those. Will do.

**Pull request merged**